src/popup/pages/MultisigDetails.vue

```diff
@@ -122,13 +131,15 @@ export default defineComponent({
     return {
       activeMultisigAccount,
       getExplorerPath,
+      openModal,
+      MODAL_CONSENSUS_INFO,
     };
   },
 });
 </script>
 
 <style scoped lang="scss">
-@use '../../styles/variables';
+@use "../../styles/variables";
```

Comment:
I don't see a reason for this change as we are using single apostrophes everywhere.